frontend/src/components/episode/episode.tsx

```diff
@@ -32,8 +33,14 @@ const Episode: React.FC = () => {
   const hasUser = Boolean(user);
 
   useEffect(() => {
+    socketService.joinRoom(id);
+    socketService.getUpdatedComments(dispatch, episodeActions.updateComments);
```

Comment:
```suggestion
```
we don't want to do it directly in the components
consider using packages for the redux to do it in the middleware